src/CMakeLists.txt

```diff
@@ -157,7 +158,7 @@ if(CMAKE_VERSION VERSION_LESS "3.1")
   endif()
   set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=gnu99")
 else()
-  set(CMAKE_CXX_STANDARD 14)
+  set(CMAKE_CXX_STANDARD 17)
```

Comment:
the C++17 support of `CMAKE_CXX_STANDARD` was introduced in cmake v3.8, see https://cmake.org/cmake/help/v3.8/prop_tgt/CXX_STANDARD.html and https://cmake.org/cmake/help/v3.7/prop_tgt/CXX_STANDARD.html, so we need to bump the required cmake version at https://github.com/ceph/ceph/pull/19657/files#diff-95e351a3805a1dafa85bf20b81d086e6R145 to `3.8`.

Reply:
All right, got it updated.